.github/labeler.yml

```diff
@@ -0,0 +1,215 @@
+documentation:
+  - docs/**/*
+  - any: [ '*.rst', '!CHANGES.rst' ]
+  - '*.md'
+  - .readthedocs.yml
+  - LICENSE
+
+installation:
+  - pyproject.toml
+  - 'setup.*'
+  - 'requirements-*.txt'
+  - MANIFEST.in
+
+ami:
+  - 'jwst/ami/**'
+
+assign_mtwcs:
+  - 'jwst/assign_mtwcs/**'
+
+assign_wcs:
+  - 'jwst/assign_wcs/**'
+
+associations:
+  - 'jwst/associations/**'
+
+background:
+  - 'jwst/background/**'
+
+barshadow:
+  - 'jwst/barshadow/**'
+
+combine_1d:
+  - 'jwst/combine_1d/**'
+
+coron3 pipeline:
+  - 'jwst/coron/**'
+  - jwst/pipeline/calwebb_coron.py
+
+cube_build:
+  - 'jwst/cube_build/**'
+
+cube_skymatch:
+  - 'jwst/cube_skymatch/**'
+
+dark_current:
+  - 'jwst/dark_current/**'
+
+datamodels:
+  - 'jwst/datamodels/**'
+
+dq_init:
+  - 'jwst/dq_init/**'
+
+engdb_tools:
+  - 'jwst/lib/engdb_*'
+
+exp_to_source:
+  - 'jwst/exp_to_source/**'
+
+extract_1d:
+  - 'jwst/extract_1d/**'
+
+extract_2d:
+  - 'jwst/extract_2d/**'
+
+firstframe:
+  - 'jwst/firstframe/**'
+
+fits_generator:
+  - 'jwst/fits_generator/**'
+
+flatfield:
+  - 'jwst/flatfield/**'
+
+fringe:
+  - 'jwst/fringe/**'
+
+gain_scale:
+  - 'jwst/gain_scale/**'
+
+group_scale:
+  - 'jwst/group_scale/**'
+
+guider_cds:
+  - 'jwst/guider_cds/**'
+
+imprint:
+  - 'jwst/imprint/**'
+
+ipc:
+  - 'jwst/ipc/**'
+
+jump:
+  - 'jwst/jump/**'
+
+lastframe:
+  - 'jwst/lastframe/**'
+
+linearity:
+  - 'jwst/linearity/**'
+
+master_background:
+  - 'jwst/master_background/**'
+
+mrs_imatch:
+  - 'jwst/mrs_imatch/**'
+
+msa_flagging:
+  - 'jwst/msaflagopen/**'
+
+outlier_detection:
+  - 'jwst/outlier_detection/**'
+
+path loss:
```

Comment:
does the pathloss label have a whitespace in the middle of it?

Reply:
yes indeed it is `path loss`, you can see here: https://github.com/spacetelescope/jwst/labels?q=path

Should the label be renamed?

Reply:
No, that's OK. I just didn't realize it was spelled that way.